packages/@okta/vuepress-site/docs/reference/api-overview/index.md

```diff
@@ -79,7 +79,34 @@ Okta will derive client request context directly from the HTTP request headers a
 
 Okta supports the standard `User-Agent` HTTP header to identify the user's browser or application. Always send a `User-Agent` string to uniquely identify your client application and version such as `Oktaprise/1.1`.
 
-> If your application is acting as a gateway or proxy, you should forward the `User-Agent` of the originating client with your API requests.
+> **Note:** If your application is acting as a gateway or proxy, you should forward the `User-Agent` of the originating client with your API requests.
+
+### Format a User-Agent string for native mobile apps
+
+Make sure that the User-Agent string that your app constructs is in the correct format. This ensures that Okta can parse the `OS` and `Browser` fields. Good User-Agent strings that can be correctly parsed by Okta contain browser information, system information, platform, platform details, and any extensions.
+
+#### Use a template to format the string
+
+We recommend that you use a template like the following to format the User-Agent string:
+
+`User-Agent: Mozilla/5.0 (<system-information>) <platform> (<platform-details>) <extensions>`
+
+#### Pass certain values in the string
+
+Alternatively, you can make sure that certain string values are present in the User-Agent string so that the OS and Browser are detected:
+
+> **Note:** Case doesn't matter since Okta converts the string to lowercase before matching.
+
+**Pass a hint about the browser**
+
+If you have a browser string such as `chrome` or `safari`, add that to the User-Agent string.
+
+**Pass operating system information**
+
+- **iOS:** Include the words `apple` or `ios` and at least one of these values: `iphone`, `ipad`, `ipod`, `ipad`.
+- **Android:** Include the words `android` or `samsung`, which infers that Android is the operating system.
+
```

Comment:
Can we add some examples here like ones specified in https://oktawiki.atlassian.net/wiki/spaces/eng/pages/900764066/Technical+Questions+from+Bank+Of+Oklahoma 
so users can see where they can be added in the string.